adapters/displayio/displayio.go

```diff
@@ -37,27 +37,18 @@ func (adapter *adapter) MakeRequests(request *openrtb2.BidRequest, requestInfo *
 	for _, impression := range request.Imp {
 		var requestExt map[string]interface{}
 
-		if impression.BidFloor == 0 {
-			errs = append(errs, &errortypes.BadInput{
-				Message: "BidFloor should be defined",
-			})
-			continue
-		}
-
-		if impression.BidFloorCur == "" {
+		if impression.BidFloorCur == "" || impression.BidFloor == 0 {
 			impression.BidFloorCur = "USD"
-		}
-
-		if impression.BidFloorCur != "USD" {
+		} else if impression.BidFloorCur != "USD" {
```

Comment:
In order to not make an unnecessary call to `requestInfo.ConvertCurrency(impression.BidFloor, impression.BidFloorCur, "USD")`, should we check for a non-zero bidFloor value first?
```
 40         if impression.BidFloorCur == "" || impression.BidFloor == 0 {
 41             impression.BidFloorCur = "USD"
 42 -       } else if impression.BidFloorCur != "USD" {
    +       } else if impression.BidFloorCur != "USD" && impression.BidFloor > 0 {
 43             convertedValue, err := requestInfo.ConvertCurrency(impression.BidFloor, impression.BidFloorCur, "USD")
 44
```